staging/src/k8s.io/apiextensions-apiserver/examples/client-go/pkg/apis/cr/register.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cr
+
+const (
+	GroupName = "cr.client-go.k8s.io"
```

Comment:
Technically it's not in client-go. Worth changing? Have no strong opinion.

Reply:
`cr.apiextensions.k8s.io`?

Reply:
cr.example.apiextensions.k8s.io? What is the CRD group? should match that.

Reply:
CRD group is `apiextensions.k8s.io`. cr.example.apiextensions.k8s.io seems good! :+1:

Reply:
:+1: